src/stereotype_client.js

```diff
@@ -51,6 +51,40 @@ class StereotypeClient {
   }
 
   /**
+   * Returns a list of JSON objects with the following fields:
+   * - templateId: string
+   * - canCopy: boolean
+   * - edit: boolean
```

Comment:
I think this field got renamed to `canEdit`. At least it should be... That's what I was commenting on the merge request for the service some time ago.

Reply:
This is completely transparent to the client but I updated the documentation and the tests.